pages/api/auth/[...nextauth].js

```diff
@@ -25,7 +25,9 @@ export default (req, res) => {
         ],
 
         // https://next-auth.js.org/schemas/adapters
-        adapter: Adapters.TypeORM.Adapter(process.env.DATABASE_URL, {
+        adapter: Adapters.TypeORM.Adapter({
+            type: 'postgresql',
+            database: process.env.DATABASE_URL,
```

Comment:
I would first propose trying to fix the bug with the env-variable formatting and resetting of the db table as mentioned in #23.

If those steps won't fix the issue: can you confirm this change fixed the issue? Right of the bat I'm getting `[next-auth][error][adapter_connection_error] MissingDriverError: Wrong driver: "postgresql" given. Supported drivers are: "cordova", "expo", "mariadb", "mongodb", "mssql", "mysql", "oracle", "postgres", "sqlite", "better-sqlite3", "sqljs", "react-native", "aurora-data-api", "aurora-data-api-pg".`

Reply:
Yes, I confirm that the above changes fixed the issue at hand. 

Given the differences that we are observing, I am guessing we may be on different versions of one of the affected packages. Since I'm the one that most recently installed and tested PR #17, I would guess I'm on the latest version.

My guess is that may be related to the `typeorm` library, here are my local versions:
```
--next-auth@3.1.0
  └── typeorm@0.2.29
```

If you look at the documentation, typeORM specifies a [different one](https://github.com/nextauthjs/adapters/tree/canary/packages/typeorm#postgresql) from [the one](https://next-auth.js.org/tutorials/typeorm-custom-models) found in nextauth.

Reply:
I confirm that the table was created anew in both my localhost and the server before I attempted any troubleshooting.

Reply:
@EkkuJ:

> If those steps won't fix the issue: can you confirm this change fixed the issue? Right of the bat I'm getting `[next-auth][error][adapter_connection_error] MissingDriverError: Wrong driver: "postgresql" given. Supported drivers are: "cordova", "expo", "mariadb", "mongodb", "mssql", "mysql", "oracle", "postgres", "sqlite", "better-sqlite3", "sqljs", "react-native", "aurora-data-api", "aurora-data-api-pg".`

I have also observed the error that you point out above, and corrected it in 4905086. Now everything works fine for me, and what you suggested did not fix the problem.